HHS/OPRE-OPS
# Ops 1187 clear button

## What changed

Added functionality to dismiss notifications.
Logged in user cannot dismiss another user's notifications.
Filter notifications by is_read status and order by creation time.

## Issue

https://github.com/HHS/OPRE-OPS/issues/1187

## How to test

Automated tests pass.

Optionally:

- Add a notification in the DB and it should appear at the top of the list.
- Click the `Clear` button to dismiss notifications.
